### PR TITLE
Cli disable machine group creation

### PR DESCRIPTION
**Pull request by Pbarbosa92**

### CODE REVIEW

File: inductiva/constants.py [hunk 1]
```diff
@@ -9,3 +9,7 @@
 TASK_KILL_MAX_API_REQUESTS = 5
 
 TASK_KILL_RETRY_SLEEP_SEC = 1.0
+
+LOADER_LOAD_PREFIX = "cmd_"
```
Comment (sergiomsantos) [inline inductiva/constants.py:13]: This is not used.It should be used in `_cli.__init__` 

```suggestion
LOADER_COMMAND_PREFIX = "cmd_"
```

File: inductiva/_cli/cmd_resources/__init__.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 """Register CLI commands for logs."""
+from ast import Constant
```
Comment (sergiomsantos) [inline inductiva/_cli/cmd_resources/__init__.py:2]: wrong import

File: inductiva/_cli/cmd_logs/__init__.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 """Register CLI commands for logs."""
 import os
+from inductiva import constants
 
 from inductiva._cli import loader, utils
```
Comment (sergiomsantos) [inline inductiva/_cli/cmd_logs/__init__.py:5]: one empty line between builtin imports and no empty lines between custom imports:

```suggestion
import os

from inductiva._cli import loader, utils
from inductiva import constants
```

File: inductiva/_cli/cmd_resources/__init__.py [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
 import os
+from inductiva import constants
```
Comment (sergiomsantos) [inline inductiva/_cli/cmd_resources/__init__.py:4]: same here

File: inductiva/_cli/cmd_resources/__init__.py (hunk 2 above)
Comment (Pbarbosa92) [reply inductiva/_cli/cmd_resources/__init__.py:2]: Removed

**Review by sergiomsantos — APPROVED** [review]

**Comment by Pbarbosa92** [comment]
**QA**

Check if adding _ at the beginning of a command or folder makes it disappears from the args.

Example:
inductiva resources -h won't show start. And the command is not recognized

@IvanPombo and @AugustoPeres

**Comment by AugustoPeres** [comment]
> **QA**
> 
> Check if adding _ at the beginning of a command or folder makes it disappears from the args.
> 
> Example: inductiva resources -h won't show start. And the command is not recognized
> 
> @IvanPombo and @AugustoPeres

QA ✅ 

I have checked and the `start` was removed from the cli:
Screenshot 2024-02-01 at 17 59 43
Screenshot 2024-02-01 at 18 00 00

**Pull request merged**